### ISSUE TITLE
Key Personnel Labor Categories

### ISSUE THREAD

**Issue by brianfunk** [18F/bpa-fs-xmas-trees#4]
# Question/Comment on the Forest Service Online Permitting Christmas Tree Module

## Name and affiliation
Brian Funk - NCI, Inc. - Agile BPA Program Manager

## Section of RFQ documents
RFQ Section 4.2.1 Categories

## Question/Comment
Can we use any labor category, within reason, for the key personnel roles listed in the RFQ?  For example, use the Digital Performance Analyst labor category for the fourth key personnel position of Writer / Content Designer / Content Strategist.

**Comment by rrefoy** [comment]
Yes, we are specifically looking for content strategy skills, as represented by the content strategy labor category. Proposals may use whichever labor category they believe best meets that skillset.
